examples/snellius_example.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#SBATCH --array=0-5
+#SBATCH -t 00:30:00
+#SBATCH -p rome
+
+#how to use this example:
+#1.clone the Picasclient github
+#git clone https://github.com/sara-nl/picasclient.git
+#cd picasclient
+#2.install packages
+#pip install picas
```

Comment:
@xinan1911 i suggest to remove the installation part from the comment here since it is covered below and in any case for non python 3.12 "pip install picas" will install an old version of picas (i think 0.2.1, when i tested without module loading the stuff below). so it might cause some confusion for some.

Reply:
i ran the script on snellius and it worked (just an extra confirmation).

looks good. tnx.